ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml

```diff
@@ -148,15 +148,10 @@ releases:
     candidate:
       product: ocp
       stream: nightly
-      version: "4.12"
+      version: "4.13"
   latest-multi:
     candidate:
       architecture: multi
-      product: ocp
-      stream: nightly
-      version: "4.12"
-  nightly-4-13:
```

Comment:
Perhaps this should move to 4.14?
@osherdp ?

Reply:
That was only being used in a few jobs and I don't know why

Reply:
Specifically `e2e-metal-assisted-kube-api-late-binding-single-node` and it's periodic

Reply:
Looks like it was just a fix we needed in 4.13: https://github.com/openshift/release/commit/4df91960b5f042c5c6b73fa5dbed9922f8e9be66

Reply:
iiuc, the only restriction is that is should be >=4.13 in the late-binding job that uses it (because otherwise we do an OS downgrade from rhcos 8.6 to rhcos 9.2 which has a non-prioritized bug on it)
see also: https://github.com/openshift/release/pull/38543